src/constants.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""This module defines constants used throughout the Synapse application."""
+
+CHECK_ALIVE_NAME = "synapse-alive"
```

Comment:
Some of these constants are only used in pebble.py. I suggest we move them there

Reply:
Related: #52